### DISCUSSION TITLE
Grid or Graph pages are not loading

### DISCUSSION THREAD

**Discussion by pavelrojkovtrepp** [apache/airflow#23608] [Q&A]
I get the exception in the attached image. Fresh install of v2.3.0 with docker. 

Thank you. 
<img width="1180" alt="Screen Shot 2022-05-09 at 9 47 30 PM" src="https://user-images.githubusercontent.com/50838778/167526457-9f9310a9-b5fd-4c9b-badb-9447980b129b.png">

**Comment by potiuk** [comment]
That's the problem with your browser's storage, not Airflow: you can find some remedies here: https://stackoverflow.com/questions/18877643/error-in-local-storage-ns-error-file-corrupted-firefox 

Worst case - remove and reinstall your browser or use another browser to verify it

**Reply by pavelrojkovtrepp** [reply]
Thank you @potiuk
